package.json

```diff
@@ -310,13 +310,20 @@
                 "title": "%AWS.command.github%",
                 "category": "AWS"
             }
+        ],
+        "jsonValidation": [
+            {
+                "fileMatch": ".aws/templates.json",
+                "url": "./out/src/schemas/templates.json"
```

Comment:
When packaged into a vsix, is the url correct?

Reply:
Yep!